pkg/autoscaler/config.go

```diff
@@ -55,19 +54,11 @@ type Config struct {
 	ScaleToZeroIdlePeriod time.Duration
 }
 
-func (c *Config) TargetConcurrency(model v1alpha1.RevisionRequestConcurrencyModelType) float64 {
-	switch model {
-	case v1alpha1.RevisionRequestConcurrencyModelSingle:
-		return c.SingleTargetConcurrency
-	case v1alpha1.RevisionRequestConcurrencyModelMulti:
-		if c.EnableVPA {
-			return c.VPAMultiTargetConcurrency
-		} else {
-			return c.MultiTargetConcurrency
-		}
-	default:
-		return c.MultiTargetConcurrency
+func (c *Config) TargetConcurrency(model v1alpha1.RevisionContainerConcurrencyType) float64 {
```

Comment:
Nit: Should the argument be called "concurrency"?

Reply:
No, I don't think so.  This isn't the concurrency limit.  It's the autoscaler's setpoint (target).